test/check-application

```diff
@@ -1628,6 +1628,42 @@ class TestApplication(testlib.MachineCase):
         b.click("#image-actions-dropdown")
         b.wait_visible("button:contains(Prune unused images).pf-m-disabled")
 
+    def testPruneUnusedImagesSystemSelections(self):
+        ''' Test the prune unused images selection options'''
+        b = self.browser
+        self.login(True)
+
+        b.click("#image-actions-dropdown")
+        b.click("button:contains(Prune unused images)")
+
+        # Deselect both
+        b.click("#deleteSystemImages")
+        b.click("#deleteUserImages")
+        b.wait_visible(".pf-c-modal-box button:contains(Prune):disabled")
+
+        # Admin / user images are selected
+        b.wait_js_func("ph_count_check", ".pf-c-modal-box__body .pf-c-list li", 6)
+        # Select user images
+        b.click("#deleteUserImages")
+        b.click(".pf-c-modal-box button:contains(Prune)")
+
+        # System images are left over
+        b.wait_js_func("ph_count_check", "#containers-images table tbody", 3)
+        checkImage(b, "quay.io/libpod/alpine:latest", "system")
+        checkImage(b, "quay.io/cockpit/registry:2", "system")
+        checkImage(b, "quay.io/libpod/busybox:latest", "system")
+
+        # Pruning again, should delete all system images
+        b.click("#image-actions-dropdown")
+        b.click("button:contains(Prune unused images)")
+        b.wait_js_func("ph_count_check", ".pf-c-modal-box__body .pf-c-list li", 3)
+        b.click(".pf-c-modal-box button:contains(Prune)")
+        b.wait_js_func("ph_count_check", "#containers-images table tbody", 0)
+
+        # Prune button should now be disabled
+        b.click("#image-actions-dropdown")
+        b.wait_visible("button:contains(Prune unused images).pf-m-disabled")
```

Comment:
Try to avoid using 'contains' as it won't work in the browser inspector.

Reply:
> as it won't work in the browser inspector.

That is correct, but I still find it nicer to user `:contains()` as it makes tests that much more readable.

Reply:
I guess it's more of a taste thing then. :whatever

Reply:
@KKoukiou do you prefer using an hardcoded id? I guess that makes things more predictable?

Reply:
This would work nicely when copying pasting css selectors to the browser inspector, but really it's not a big deal.